package.json

```diff
@@ -27,10 +27,12 @@
     "start": "npm run dev",
     "prebuild": "rm -rf dist",
     "lint": "eslint .",
-    "e2e": "playwright test",
-    "e2e:debug": "playwright test --project chromium --headed",
+    "e2e": "npm run lint && playwright test",
+    "e2e:debug": "npm run lint && playwright test --project chromium --headed",
```

Comment:
I'd remove `lint` from `e2e` tests and add a new `test` script that run `lint` and `e2e`.

Reply:
Changed